### DISCUSSION TITLE
Active window screenshots

### DISCUSSION THREAD

**Discussion by salahoued** [labwc/labwc#2022] [Q&A]
Hello,
For the time being I'm using `grim/slurp` or `wayshot/waysip` to screenshot a region,
Is there any way to take a screenshot of the active window directly without `slurp/waysip`?
Or a way to get active window geometry for `grim/wayshot` without user input?

**Comment by Consolatis** [comment]
Not at this point.

In the longer term it should be possible via the ext-screencapture protocol.
Labwc \*might\* also implement a manual window screenshot ability at some point but there is no work related to that as of now.

Edit:
Depending on the use-case there is a workaround in spawning a nested compositor and then start the application maximized within there and do a screenshot. The usefulness of that approach really depends on the use-case though.
